service/chartconfig/v1/helm/helm.go

```diff
@@ -44,6 +44,16 @@ func New(config Config) (*Client, error) {
 	return newHelm, nil
 }
 
+// DeleteRelease uninstalls a chart given its release name
+func (c *Client) DeleteRelease(releaseName string, options ...helmclient.DeleteOption) error {
```

Comment:
Thats the exact name as in the helm package. Is that fine? Cause the other functions have different names than in the package.

Reply:
I think using the same name is fine. We do that for ReleaseStatus and ReleaseContent. InstallTarball is different because it uses the tarball we download from CNR.